### PR TITLE
Delete all complete

### PR DESCRIPTION
**Pull request by jubenjam**
Button to delete all completed tasks at once

### CODE REVIEW

**Review by yuneihlaing — APPROVED** [review]
Looks good!

File: react-frontend/src/MyApp.js [hunk 1]
```diff
@@ -232,6 +217,20 @@ function MyApp(props) {
     }
   }
 
+  function removeAll() {
+    removeComplete().then((result) => {
+      if (result && result.status === 204) {
```
Comment (CameronBill) [inline react-frontend/src/MyApp.js:222]: Characters seems a little confusing, is this the list of task. We should maybe refactor/rename this

File: expressjs-backend/models/task-services.js [hunk 2]
```diff
@@ -55,6 +55,15 @@ async function findTaskById(id) {
   }
 }
 
+async function deleteCompeteByUser(user) {
```
Comment (CameronBill) [inline expressjs-backend/models/task-services.js]: Should this say deleteCompleteByUser? Looks like it works, not a big deal

**Review by CameronBill — APPROVED** [review]
Couple of function/variable names that maybe should be renamed. Shouldn't have caused any functionality issues. Overall LGTM

File: expressjs-backend/models/task-services.js (hunk 2 above)
Comment (jubenjam) [reply expressjs-backend/models/task-services.js]: yes it should. thanks for catching that

File: react-frontend/src/MyApp.js (hunk 1 above)
Comment (jubenjam) [reply react-frontend/src/MyApp.js:222]: That makes sense, though that can be done in a different pull request.

File: expressjs-backend/models/task-services.js [hunk 3]
```diff
@@ -172,4 +185,5 @@ exports.editTask = editTask;
 exports.getCategories = getCategories;
 exports.getCategoriesOfUser = getCategoriesOfUser;
 exports.completeTask = completeTask;
+exports.deleteCompeteByUser = deleteCompeteByUser;
```
Comment (arigoodmann22) [inline expressjs-backend/models/task-services.js]: did you catch this typo too?

**Review by arigoodmann22 — APPROVED** [review]

File: expressjs-backend/models/task-services.js (hunk 3 above)
Comment (jubenjam) [reply expressjs-backend/models/task-services.js]: Fixed that one when Cameron mentioned the other typo. Also fixed the one in backend.js, I'll push those changes soon

**Review by dustint121 — APPROVED** [review]
LGTM

**Pull request merged**